runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/OpAdapter.java

```diff
@@ -51,20 +56,28 @@
   private static final Logger LOG = LoggerFactory.getLogger(OpAdapter.class);
 
   private final Op<InT, OutT, K> op;
+  private final String opName;
```

Comment:
It's not opName. it's transformFullName, right? Can we at least make the name correcct?